tests/cloner_test.go

```diff
@@ -2610,6 +2619,14 @@ var _ = Describe("all clone tests", func() {
 			size := "1Gi"
 			createSnapshot(size, nil, volumeMode)
 
+			dataVolume := utils.NewDataVolumeForCloningFromSnapshot("pass-all-labels-and-annotations-test", size, snapshot.Namespace, snapshot.Name, nil, &volumeMode)
+			dataVolume.Labels = map[string]string{"test-label-1": "test-label-value-1"}
+			dataVolume.Annotations = map[string]string{"test-annotation-1": "test-annotation-value-1"}
+			dataVolume, err = utils.CreateDataVolumeFromDefinition(f.CdiClient, targetNs.Name, dataVolume)
+			// All labels and annotations passed
+			Expect(dataVolume.Labels["test-label-1"]).To(Equal("test-label-value-1"))
+			Expect(dataVolume.Annotations["test-annotation-1"]).To(Equal("test-annotation-value-1"))
```

Comment:
You added labels and annotations to the DV, so you need to check they eere passed  to the **PVC**.